test/test_generator.py

```diff
@@ -95,30 +115,33 @@ def test_generation_level_3(tmpdir):
 data = {{
   "url": "URL_OF_STORED_AUDIO_FILE"
 }}
-response = requests.post(url, headers=headers, files=file, data=data)
-print(response.text)
-Summarize the text.
-Create an audio file of the summarized text.
+response = requests.post(url, headers=headers, data=data)
+assert response.status_code == 200
+print('This is the text from the audio file:', response.json()['text'])
+2. Summarize the text (~50 words) while still maintaining the key facts.
+3. Create an audio file of the summarized text using a tts library.
+4. Return the the audio file as base64 encoded binary.
+Example input file: https://www.signalogic.com/melp/EngSamples/Orig/ENG_M.wav
 ''',
         str(tmpdir) + 'microservice',
-        'gpt-3.5-turbo'
+        'gpt-4'
```

Comment:
Can turbo not solve the issue?

Reply:
yes, it was not able, but I can try again with the recent changes

Reply:
I was just curious. Let's merge!